src/services/edusharing/logic/connector.js

```diff
@@ -182,8 +187,9 @@ class EduSharingConnector {
 		}
 
 		const urlBase = `${Configuration.get('ES_DOMAIN')}${ES_PATH.SEARCH}?`;
-		const url = urlBase
-			+ [
+		const url =
```

Comment:
Codacy found an issue: [There should be no line break before or after '='.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045787)